homework-2/task-1.py

```diff
@@ -0,0 +1,4 @@
+isTrue=input("Enter true or false: ") == "true"
```

Comment:
პითონში ვწერთ snake_case
```suggestion
is_true=input("Enter true or false: ") == "true"
```